### PR TITLE
Fix scroll draw on Safari (deploy)

### PR DESCRIPTION
**Pull request by Techno-coder**
Cherry pick of #80.

### CODE REVIEW

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -109,11 +109,11 @@ a.progress_link:hover {
 
 #main {
   top: 30px;
-  left: 200px;
+  margin-left: 200px;
   right: 0;
   bottom: 0;
-  position: fixed;
   overflow-y: auto;
+  outline: none;
```
Comment (puqeko) [inline app/assets/stylesheets/style.css]: What is the function of adding `outline: none`?

Comment (Techno-coder) [reply app/assets/stylesheets/style.css]: Hovering or clicking inside the main container (problem statement area) creates a "selected" outline.

Comment (puqeko) [reply app/assets/stylesheets/style.css]: The default behaviour of outline should be none by default. This seems to be a consequence of the `tabindex="-1"` attribute. The only two reasons I can think of for needing `tabindex="-1"` is for accessibility and to facilitate automated tests. As far as accessibility, removing it doesn't seem to change the tab order of the page. If coveralls returns the same result, I would be inclined to remove the tabindex attribute instead

Comment (Techno-coder) [reply app/assets/stylesheets/style.css]: Per mirrored commit on https://github.com/NZOI/nztrain/pull/80, coveralls remains unchanged.

**Comment by puqeko** [comment]
Deployed and restarted

Note: the main window appears to have moved 1-2 pixels with no obvious consequences.

**Comment by tom93** [comment]
This PR was reverted (by force-pushing the parent commit to the 'deployed' branch) because of an issue reported by @puqeko in chat.

> If you scroll up and down the menu bar stops sticking to the top and leaves a white bar (on macOS, using Opera, Safari, and Chrome but not Firefox).
> Some of that behaviour was preexisting but it was much more noticeable/pronounced after the change.

screenshot

**Pull request merged**